### DISCUSSION TITLE
When k8s deploys doris, can its pvc use aws s3?

### DISCUSSION THREAD

**Discussion by zan-liu** [apache/doris#25811] [A - General / Q&A]
When k8s deploys doris, can its pvc use aws s3? I haven't seen any documentation on this practice. Can anyone give me an answer?

**Comment by intelligentfu** [comment]
Doris does not care about the backend PV provider of PVC. In my opinion, S3 has provided a CSI plugin. Selectdb provides k8s-operator for deploying Doris on k8s, which you can experience.
